hivemind/p2p/p2p_daemon.py

```diff
@@ -68,10 +63,12 @@ def __init__(self):
     @classmethod
     async def create(cls, *args, quic: bool = True, tls: bool = True, conn_manager: bool = True,
```

Comment:
param *args appears to be unused throughout P2P.create calls
```suggestion
    async def create(cls, *, quic: bool = True, tls: bool = True, conn_manager: bool = True,
```

Reply:
They are used in the `self._make_process_args()` call.